docs/test/Runtime 0.1.ipynb

```diff
@@ -0,0 +1,562 @@
+   "outputs": [],
+   "source": [
+    "# Load default legacy account\n",
+    "service = IBMRuntimeService(auth=\"legacy\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bfcc2cec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load legacy account with name\n",
+    "service = IBMRuntimeService(name=\"my-legacy-account\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76d4c05a",
+   "metadata": {},
+   "source": [
+    "### Enable Legacy Account for current session without saving"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ec0980d9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable legacy account with auth and token parameters\n",
+    "service = IBMRuntimeService(auth=\"legacy\", token=\"xyz\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f76d6841",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable legacy account with token and url parameters\n",
+    "service = IBMRuntimeService(token=\"xyz\", url=\"https://auth.quantum-computing.ibm.com/api\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bf082bf",
+   "metadata": {},
+   "source": [
+    "### Enable Legacy Account using Env variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c81d66c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Env variables take preference over saved credentials\n",
+    "QISKIT_IBM_API_TOKEN=xyz\n",
+    "QISKIT_IBM_API_URL=https://auth.quantum-computing.ibm.com/api\n",
+    "QISKIT_IBM_INSTANCE=qiskit-hub-1/qiskit-group-1/qiskit-project-1\n",
+    "\n",
+    "# Load legacy account from environment variables\n",
+    "service = IBMRuntimeService()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11908543",
+   "metadata": {},
+   "source": [
+    "### Delete Legacy Account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "056dafef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Delete default legacy account\n",
+    "IBMRuntimeService.delete_account(auth=\"legacy\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb530615",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Delete legacy account with name\n",
+    "IBMRuntimeService.delete_account(name=\"my-legacy-account\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "471e1741",
+   "metadata": {},
+   "source": [
+    "## Cloud\n",
+    "Using IBM Cloud API key and CRN.\n",
+    "\n",
+    "### Save Cloud Account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4e63a883",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save default cloud account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    auth=\"cloud\", \n",
+    "    token=\"xyz\", \n",
+    "    url=\"https://cloud.ibm.com\",\n",
```

Comment:
This proposal assumes `https://cloud.ibm.com` but the current [implementation](https://github.com/Qiskit/qiskit-ibm-runtime/blame/0b6deb742d41153252d9114a0866c0c3e0cf8196/qiskit_ibm_runtime/accounts/account.py#L27) expects `https://us-east.quantum-computing.cloud.ibm.com`.

I would vote for sticking to the proposed URL in the design document and derive all other required API URLs from it (e.g. `https://resource-controller.cloud.ibm.com`, `https://us-east.quantum-computing.cloud.ibm.com`, ...).

Reply:
How do you plan on deriving that from `https://cloud.ibm.com`? For example, in the future my account might have both `us-east` and `us-south`, but I prefer using `us-east` since that's closer to me. How would the code know about my preference?

Reply:
It appears to me that the location specific API endpoint can be derived from the Cloud Resource Name (CRN), e.g.
```
crn:v1:bluemix:public:quantum-computing:us-east:a/41d9bde3ef128894870dfbeaca4a5260:0ddc92a3-00be-4ec8-987e-cafda1b086fb:: 
-> 
https://us-east.quantum-computing.cloud.ibm.com
```

With https://github.com/Qiskit/qiskit-ibm-runtime/issues/44 we will leverage the global [resource controller API](https://cloud.ibm.com/apidocs/resource-controller/resource-controller#list-resource-instances) that also returns this information if users pass in a service name instead of the CRN via the `instance` parameter.

Reply:
I personally don't like inferring the url based on the CRN either, since we have 0 control over the relationship between the two (not that it's likely to change), unless there is an official API that'd do the translation for us.

Reply:
There's no official API as far as I can tell but the CRN format in general is fixed and [publicly documented](https://cloud.ibm.com/docs/account?topic=account-crn#format-crn). 

My feeling is it's more convenient for users if they do not need to pass in information manually that we can infer internally.

But let's assume for a bit that we will stick with `https://us-east.quantum-computing.cloud.ibm.com`.
- How do you suggest should we infer the resource controller API URL? Would this be an additional parameter users need to pass?

Reply:
ok if the CRN format is fixed, I suppose it's not so bad to infer the url from it. 

> How do you suggest should we infer the resource controller API URL? Would this be an additional parameter users need to pass?

I'd prefer if we don't add yet another parameter, given how many we already have. I suppose we'll need to hard code this one 😞 

All this inference / hard coding should only apply if the url is `https://cloud.ibm.com`.